builder/xenserver/common/step_type_boot_command.go

```diff
@@ -114,9 +115,9 @@ func (self *StepTypeBootCommand) Run(ctx context.Context, state multistep.StateB
 		return multistep.ActionHalt
 	}
 
-	ui.Say(fmt.Sprintf("Received response: %s", string(buffer)))
+	ui.Message(fmt.Sprintf("Received response: %s", string(buffer)))
 
-	vncClient, err := vnc.Client(tlsConn, &vnc.ClientConfig{Exclusive: true})
+	vncClient, err := vnc.Client(tlsConn, &vnc.ClientConfig{Exclusive: !config.PackerDebug})
```

Comment:
Can you explain the rationale for setting this value based on `config.PackerDebug`? Is the idea that you'd like to snoop on the VNC connection?

Reply:
Hello @ddelnano sorry for the delay. The idea behind this proposal is to be able to see what is typed in VNC when there is a problem with the construction of a machine.

For example, we have XCP-NG and XO installed, I want to build a specific template. I start by creating my template in debug mode to see all the steps and I wanted to watch the console on XO to have a simple look at what is happening in VNC. With this feature, I can adapt wait markers of the boot command

This is a debug mode proposal because I don't want VNC to be accessible when it's in production mode with CI/CD.

What do you think? I can revise my PR to create a specific setting if you prefer

Reply:
Thanks for confirming! Just wanted to make sure I understood the utility of it!